app.py

```diff
@@ -2,7 +2,7 @@
 import os
 from flask import Flask, jsonify, request
 import requests
-from flask_cors import CORS
+from flask_cors import CORS, cross_origin
```

Comment:
Is this really all we needed to add here??

Reply:
Surprisingly yes!  I was amazed how little went into getting it set up.  Fortunately Alex caught the `cross_origin` addition which pulled everything together into a working state.

Reply:
Right on! Awesome job here.